packages/ap-audit/docs/api.rst

```diff
@@ -0,0 +1,51 @@
+API
+=============================================
+
+
+Audit decorator
+***************
+
+.. autofunction:: alliance_platform.audit.create_audit_model_base
+
+Events
+******
+
+.. autoclass:: alliance_platform.audit.events.PatchedEvent
+.. autoclass:: alliance_platform.audit.events.AuditSnapshot
+.. autofunction:: alliance_platform.audit.events.create_event
+
+Views
+*****
+
+.. autoclass:: alliance_platform.audit.api.AuditLogView
+
+    .. automethod:: get_single_queryset
+    .. automethod:: get_multiple_queryset
+    .. automethod:: get_queryset
+
+.. autoclass:: alliance_platform.audit.api.AuditUserChoicesView
+
+.. autoclass:: alliance_platform.audit.api.EventUnion
+    :members:
+
+Registry
+********
+
+.. autoclass:: alliance_platform.audit.registry.AuditRegistry
+    :members:
+
+Search
+******
+
+.. autofunction:: alliance_platform.audit.search.search_audit_by_context
+
+Middleware
+**********
+
+.. autoclass:: alliance_platform.audit.middleware.AuditMiddleware
+
+
+Templatetags
+************
+
+.. autofunction:: alliance_platform.audit.templatetags.alliance_platform.audit.render_audit_list
```

Comment:
using autofunction for templatetags sucks :( so now templatetags docs should be written like:

```
.. templatetag:: render_audit_list
                                                                                                                     
``render_audit_list``
---------------------

description here... duplication from the tag itself unfortunately (or you can have a dev friendly header doc
in the templatetag and user friendly docs here)
                                  
================= =============================================================
Argument          Description
================= =============================================================
``arg1``          blah blah
``**kwargs``      Any other props to pass through to the audit component
================= =============================================================                                                                                             
```

(you can see examples in templatetags.rst in frontend package)

You can then link to templatetags with just the tagname like 

```
:ttag:`render_audit_list`
```